compass/landice/tests/antarctic/default/__init__.py

```diff
@@ -0,0 +1,28 @@
+from compass.testcase import TestCase
+from compass.landice.tests.antarctic.mesh import Mesh
+
+
+class Default(TestCase):
```

Comment:
I think we should rename this to something more descriptive.  Eventually we may have a handful of AIS mesh generation test cases and possibly various tests for running AIS simulations.  How about we rename this to `lowResMesh` or maybe more generically `meshGen` if we think we will support multiple resolutions for the same mesh generation test case (which I think will be the case).

Reply:
I'm fine with this.  To be clear, it would need to be `LowResMesh` or `MeshGen` (starting with a capital letter).

Reply:
I think `MeshGen` is better because low res is just the default configuration of this test case.